### DISCUSSION TITLE
fcitx5-rime 不可用，虽然仓库里有

### DISCUSSION THREAD

**Discussion by xzl01** [linuxdeepin/developer-center#3607] [Troubleshooting | 故障排查]
> 已经使用命令行装上了，也重启输入法框架或系统了，都不可用。
  为什么 fcitx5-rime 正常使用呢？
  求适配。

https://bbs.deepin.org/post/251172
来自论坛反馈，希望验证并修复

**Comment by xzl01** [comment]
@asterwyx

**Comment by yanhuacuo** [comment]
物理装机 deepin V23 Alpha 2 ，实测确实不可用。

**Reply by Decodetalkers** [reply]
> 物理装机 deepin V23 Alpha 2 ，实测确实不可用。

找到了，添加输入法，Chinese unknown , rime在那里

**Comment by 98wb** [comment]
我在华为笔记本上物理装机了一下昨天发布的 V23 Alpha 2 ，按照论坛里指点添加了 Chinese unknown ，但是依然缺失 fcitx5 框架本就存在的主题配置界面，也缺少 fcitx5 原生支持的高级设置页，不能改候选字体与大小，更不用说配色等等了。

希望把 fcitx5 框架固有的设置界面移植过来，这些本不需要重构吧？


- ![pic](https://github.com/98wb/win/raw/master/2023-02-09_094039.png)

- ![pic](https://github.com/98wb/win/raw/master/2023-02-09_094205.png)

**Reply by Decodetalkers** [reply]
> 我在华为笔记本上物理装机了一下昨天发布的 V23 Alpha 2 ，按照论坛里指点添加了 Chinese unknown ，但是依然缺失 fcitx5 框架本就存在的主题配置界面，也缺少 fcitx5 原生支持的高级设置页，不能改候选字体与大小，更不用说配色等等了。
> 
> 希望把 fcitx5 框架固有的设置界面移植过来，这些本不需要重构吧？
> 
>     * ![pic](https://github.com/98wb/win/raw/master/2023-02-09_094039.png)
> 
>     * ![pic](https://github.com/98wb/win/raw/master/2023-02-09_094205.png)

设置中心，键盘，输入法，添加输入法，Chinese unknown , rime在那里
控制中心有fcitx5插件，我们做了

fcitx5-config-qt版本太低，会尽快升级的，当然插件也要维护

**Reply by yanhuacuo** [reply]
> > 我在华为笔记本上物理装机了一下昨天发布的 V23 Alpha 2 ，按照论坛里指点添加了 Chinese unknown ，但是依然缺失 fcitx5 框架本就存在的主题配置界面，也缺少 fcitx5 原生支持的高级设置页，不能改候选字体与大小，更不用说配色等等了。
> > 希望把 fcitx5 框架固有的设置界面移植过来，这些本不需要重构吧？
> > ```
> > * ![pic](https://github.com/98wb/win/raw/master/2023-02-09_094039.png)
> > 
> > * ![pic](https://github.com/98wb/win/raw/master/2023-02-09_094205.png)
> > ```
> 
> 设置中心，键盘，输入法，添加输入法，Chinese unknown , rime在那里 控制中心有fcitx5插件，我们做了
> 
> fcitx5-config-qt版本太低，会尽快升级的，当然插件也要维护

找到了，添加成功了。不过确实当前这个样子，有点不友好，希望改进， Deepin用户 苦 Fcitx4 久矣 :)

**Comment by cppxiaozhu** [comment]
中州韵用户关注中，借宿在 RIME 下的小鹤用户、双拼用户、五笔用户、郑码用户等等，还是很多的。希望做好 Fcitx5-Rime 的移植，特别是 Fcitx5 的主题系统：

比如下图实现的「形码拆分」，是靠 Fcitx5 的主题系统指定「矢量字体」实现的，而该「矢量字体」则由不同的用户群自己制作，需要特别指定，因为很多汉字的结构不是现成的字，这些「矢量字体」是通过映射 PUA 点位完成的。

![拆分](https://98wb.github.io/img/1108/10.png)

另一方面，较新的 Fcitx5 设置界面，其实是吸纳了众多不同形码用户的需求，综合改进的。这些设置界面提供的选项对我们是非常重要的：

![设置](https://98wb.github.io/img/1108/12.png)

一直没有使用 Deepin，也没有办法在自己的社区推广 Deepin，因为输入法框架就是很大的困扰，既然引入了 Fcitx5，那就做一套完善的移植计划吧。

关注中……

**Comment by Zeno-sole** [comment]
@asterwyx  更新一波 ？

**Reply by asterwyx** [reply]
> @asterwyx 更新一波 ？

可以

**Comment by ellicefix** [comment]
嚯，fcitx5-rime 终于进入 Deepin 了么，有生之年，蹲个更新。

**Comment by cppxiaozhu** [comment]
蹲个更新

**Comment by asterwyx** [comment]
@zsien 关注一下

**Comment by yanhuacuo** [comment]
蹲个进度

**Comment by zsien** [comment]
其实目前是可以用的，只不过分组在「Chinese - unknown」下，直接从里面添加中州韻即可。
分组的问题得处理处理

**Reply by cppxiaozhu** [reply]
> 其实目前是可以用的，只不过分组在「Chinese - unknown」下，直接从里面添加中州韻即可。 分组的问题得处理处理

不仅是「分组」有问题啊，楼上提到的 fcitx5 框架自带的界面设置工具都是缺失的：

- fcitx5 输入方案可选项的调整界面是缺失的

![pic](https://camo.githubusercontent.com/095ab738a37eb4a6767f15f6dce386be067df58ce0e4b7ecd86c8ccc595d5f92/68747470733a2f2f393877622e6769746875622e696f2f696d672f313130382f31322e706e67)


- fcitx5 自带的附加组件设置界面是缺失的

![pic](https://raw.githubusercontent.com/98wb/win/master/2023-02-09_094039.png)


- fcitx5 自带的设置界入口，也是缺失的
![pic](https://raw.githubusercontent.com/98wb/win/master/2023-02-09_094205.png)


## 另外，fcitx5 的主题控制界面也是缺失的。

**Reply by zsien** [reply]
这个在更新了，fcitx5 自带的设置界面，所有的选项都会回来，跟上游完全一样

**Reply by cppxiaozhu** [reply]
> 这个在更新了，fcitx5-configtool 所有的选项都会回来，跟上游完全一样

那太棒了，Fcitx5 框架的设置界面里，有个「skin」配置页，可以调整主题，非常方便。

分享一下我自己调整的界面，几乎与 MacOS 上的「鼠须管」外观一致，很舒服，fcitx5 在外观方面的进步太大了，v23 引入真是皆大欢喜：

![pic](https://raw.githubusercontent.com/cppxiaozhu/00/main/72668d336975a960623f6a4ccf27df86.png)

**Reply by zsien** [reply]
挺好看（比 macOS 默认大气多了

**Comment by yanhuacuo** [comment]
蹲个进度

**Reply by BLumia** [reply]
其实不用 bump thread，我们有在线 Matrix 实例，相关话题也会在 `#deepin-community:deepin.org` （通用的开发者群）和 `#deepin-sig-sysdev-team:matrix.org` （系统包的维护讨论之类）进行，有兴趣可以来关注/参与讨论甚至直接贡献/维护软件包 :)

**Comment by asterwyx** [comment]
已修复#3610，集成中#3734。

**Reply by yanhuacuo** [reply]
DeepinV23 何时收到这些更新推送呢？

**Reply by asterwyx** [reply]
测试完毕后，V23可立即以内测的方式收到该更新推送。

**Comment by asterwyx** [comment]
测试完毕，fcitx5-rime目前仍在unknown分类下，会在下次更新的时候集成（已有提交修复）。
https://github.com/linuxdeepin/developer-center/issues/3734#event-8634882851
